op/status.go

```diff
@@ -51,9 +51,15 @@ func GetNodeStatus(ctx context.Context, inf cke.Infrastructure, node *cke.Node,
 		return nil, err
 	}
 
+	isAddedmember, err := ce.VolumeExists(EtcdAddedMemberVolumeName)
+	if err != nil {
+		return nil, err
+	}
+
 	status.Etcd = cke.EtcdStatus{
 		ServiceStatus: ss[EtcdContainerName],
 		HasData:       etcdVolumeExists,
+		IsAddedMember: isAddedmember,
```

Comment:
How about changing `HasData` to `etcdVolumeExists && isAddedmember`
and not adding `IsAddedMember` field?

Reply:
It sounds good. I'll refactor it.